database/insertData/03_template_D_basicReviewTesting.js

```diff
@@ -269,6 +269,7 @@ exports.queries = [
                 level: 1
                 restrictions: { templateSectionRestrictions: ["S2"] }
               }
+              { permissionNameId: 10500, restrictions: { canSelfAssign: true }, stageNumber:1, level: 1 }
```

Comment:
Just wondering why `canSelfAssign` needs to be a distinct restriction. Surely having review permission AND assign permissions for a given template means that user can self-assign? Or, if you want more fine-grained (i.e. can ONLY assign themselves, not someone else), wouldn't you just add a more specific "selfAssign" policy and connect it to a "selfAssign" permission name?